### PR TITLE
Draft 11 updates

### PR DESCRIPTION
**Pull request by ekr**
Here are some missing -11 changes that turned up during my interop with @kazuho

### CODE REVIEW

File: packet.go [hunk 1]
```diff
@@ -203,7 +203,7 @@ func (p packetHeader) Version__length() uintptr {
 	return 0
 }
 
-func newPacket(pt packetType, destCid ConnectionId, srcCid ConnectionId, ver VersionNumber, pn uint64, payload []byte) *packet {
+func newPacket(pt packetType, destCid ConnectionId, srcCid ConnectionId, ver VersionNumber, pn uint64, payload []byte, tagLen int) *packet {
```
Comment (martinthomson) [inline packet.go]: #69 covers this

File: packet.go [hunk 2]
```diff
@@ -106,10 +106,10 @@ type packetHeader struct {
 	// Consult getHeaderType if you want a value that corresponds to the
 	// definition of packetType.
 	Type                    packetType
+	Version                 VersionNumber
```
Comment (martinthomson) [inline packet.go]: Why move this?

File: crypto.go [hunk 3]
```diff
@@ -22,8 +22,8 @@ func infallibleHexDecode(s string) []byte {
 
 var kQuicVersionSalt = infallibleHexDecode("9c108f98520a5c5c32968e950e8a2c5fe06d6c38")
 
-const clientCtSecretLabel = "client hs"
-const serverCtSecretLabel = "server hs"
+const clientCtSecretLabel = "client in"
+const serverCtSecretLabel = "server in"
```
Comment (martinthomson) [inline crypto.go]: This isn't -11.

Comment (ekr) [reply crypto.go]: OK.

File: packet.go (hunk 2 above)
Comment (ekr) [reply packet.go]: Things are serialized in the order they appear in the data structure and VN is the second value. See: 

https://tools.ietf.org/html/draft-ietf-quic-transport-11#section-4.1

File: packet.go (hunk 1 above)
Comment (ekr) [reply packet.go]: I don't see a relevant change in 69, which doesn't seem to affect packet.go at all

**Comment by ekr** [comment]
Merged by hand

**Pull request closed without merging**